ansible/roles/minidlna/templates/etc/ansible/facts.d/minidlna.fact.j2

```diff
@@ -0,0 +1,9 @@
+{# Copyright (C) 2021 Julien Lecomte <julien@lecomte.at>
```

Comment:
It would be better to make the fact a Python script. That way users can use `run-parts /etc/ansible/facts.d` to get a nice overview, or other services can benefit from this. The `postwhite` role has a nice example of a simple Python fact script.

Reply:
I'll look into that.